server.js

```diff
@@ -5,18 +5,25 @@ const session = require('express-session');
 const passport = require('./config/passport');
 
 // Setting up port and requiring models for syncing
-const PORT = process.env.PORT || 8080;
 const db = require('./models');
+const PORT = process.env.PORT || 8080;
+// const MySQLStore = require('express-mysql-session')(session);
```

Comment:
Can this be removed since we assign a mysqlStore property in the db object?

Reply:
Done